### PR TITLE
Update rockset version

### PR DESCRIPTION
**Pull request by clee2000**
upgrading rockset to 1.0.3

the diff looks like it gets rid of dependency on six but i think python-dateutils still uses it but is better about downloading it

### CODE REVIEW

File: .github/scripts/fetch_latest_green_commit.py [hunk 1]
```diff
@@ -39,22 +39,33 @@ def get_latest_commits() -> List[str]:
 
     return commits
 
-def query_commits(commits: List[str], qlambda: Any) -> Any:
-    params = rockset.ParamDict()
-    params['shas'] = ",".join(commits)
-    results = qlambda.execute(parameters=params)
+def query_commits(commits: List[str]) -> List[Dict[str, Any]]:
+    rs = rockset.RocksetClient(
+        host="api.usw2a1.rockset.com", api_key=os.environ["ROCKSET_API_KEY"]
```
Comment (huydhn) [inline .github/scripts/fetch_latest_green_commit.py:44]: Different endpoints for the new Rockset API?

**Review by huydhn — APPROVED** [review]
LGTM! Just curious about the benefit of the newer version, perf gain?

File: .github/scripts/fetch_latest_green_commit.py (hunk 1 above)
Comment (clee2000) [reply .github/scripts/fetch_latest_green_commit.py:44]: not entirely sure, but followed https://github.com/rockset/rockset-python-client#migration-guide and https://rockset.com/docs/rest-api/ to make these changes

**Comment by clee2000** [comment]
> LGTM! Just curious about the benefit of the newer version, perf gain?

no, its just that i switched to 1.0.3 in my changes to trymerge and got annoyed at switching virtual envs all the time...

**Pull request closed without merging**